docs/source/development/linting.md

```diff
@@ -0,0 +1,84 @@
+# Linting
+
+## Introduction
+Lint, or a linter, is a static code analysis tool used to flag programming errors, bugs, stylistic errors and suspicious constructs.
```

Comment:
This line refers specifically to the software 'Lint' (https://en.wikipedia.org/wiki/Lint_(software)), so I don't think it is appropriate to use this here.

I also don't think that 'suspicious constructs' makes too much sense in the context of Python.

```suggestion
Linting tools are used to improve the quality of your code by checking for errors and making sure it meets a stylistic standard before it is run.
```